diff --git a/config.def.h b/config.def.h
index 1edb647..7850977 100644
--- a/config.def.h
+++ b/config.def.h
@@ -10,8 +10,8 @@ static const char *prompt      = NULL;      /* -p  option; prompt to the left of
 static const char *colors[SchemeLast][2] = {
 	/*     fg         bg       */
 	[SchemeNorm] = { "#bbbbbb", "#222222" },
-	[SchemeSel] = { "#eeeeee", "#005577" },
-	[SchemeOut] = { "#000000", "#00ffff" },
+	[SchemeSel] = { "#eeeeee", "#7c685e" },
+	[SchemeOut] = { "#000000", "#cab0a1" },
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines      = 0;
